stylus/components/modals.styl

```diff
@@ -40,19 +49,34 @@ $modals
         flex-direction    column
 
     .coz-modal--small
-        max-width 34rem
+        width small
+
+        @media (max-width: small + 1rem)
+            width auto
 
     .coz-modal--medium
-        max-width 36rem
+        width medium
+
+        @media (max-width: medium + 3rem)
```

Comment:
Where does the 3 come from ? Wouldn't it be easier to have a loop here to output all this code that is very similar.

Reply:
It's the padding of the wrapper. I guess I could use variables here too, but as for the loop, I'm not sure it would be easy to do since sizes and paddings are very contextual.

Reply:
And the 6rem below ?

A variable to describe the intention would be good I guess. It is difficult to understand when there are magic numbers.

Reply:
true true. 6rem = 2*3rem of padding.

I'll use variables and make the math easy to read.